cognite/client/data_classes/capabilities.py

```diff
@@ -174,6 +335,11 @@ class UnknownScope(Capability.Scope):
     def __getitem__(self, item: str) -> Any:
         return self.data[item]
 
+    def is_within(self, other: Self) -> bool:
+        if isinstance(other, AllScope):
+            return True
+        raise NotImplementedError("Unknown scopes cannot be compared")
```

Comment:
We don't know if the unknown scope has an AllScope, so I suggest we just always raise here